server/src/utilities/processFile/processFile.ts

```diff
@@ -0,0 +1,31 @@
+import { readFile } from 'fs';
+import Project from 'ts-morph';
+
+import MezzuriteComponent from '../../models/MezzuriteComponent';
+import getComponentType from '../getComponentType';
+import generateComponent from '../generateComponent';
+
+function processFile (filePath: string, project: Project): Promise<MezzuriteComponent> {
```

Comment:
There's no null check for the `project` parameter. If it's `null` or `undefined`, it won't break until line 16, which is after spending some resources in reading the file and checking its component type.

Reply:
Resolved via https://github.com/Microsoft/vscode-mezzurite/pull/31/commits/792b7100303dcf4fea87143696289b1e87c7d5e4.